nservicebus/messaging/command-event-comparison-table.include.md

```diff
@@ -0,0 +1,9 @@
+|   | Commands | Events |
+|---|:--------:|:------:|
+| Marker Interface | `ICommand` | `IEvent` |
+| Logical Senders | Many | 1 |
```

Comment:
@indualagarsamy I think what you're saying about receivers publishing is confusing. :wink: What I'm really trying to get at here is the rules for a *single* event. So yeah, when an endpoint receives a message, it can turn around and publish something else. But if I'm talking about the OrderPlaced event, it has one logical sender and multiple logical receivers. Can you help me make that more clear?